fitbenchmarking/controllers/levmar_controller.py

```diff
@@ -54,9 +55,10 @@ def setup(self):
             lb = [None if x == -np.inf else x for x in lb]
             ub = [None if x == np.inf else x for x in ub]
             self.param_ranges = list(zip(lb, ub))
+        self.lm_y = np.zeros(self.data_y.shape)
 
     # pylint: disable=unused-argument
```

Comment:
This pylint disable warning statement can be removed now as the unused argument, x, has been removed